clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/ops/CopyLastGoogleServerGroupAtomicOperation.groovy

```diff
@@ -47,6 +47,9 @@ class CopyLastGoogleServerGroupAtomicOperation implements AtomicOperation<Deploy
   BasicGoogleDeployHandler basicGoogleDeployHandler
 
   @Autowired
+  GoogleClusterProvider googleClusterProvider
+
+  @Autowired
   GoogleSecurityGroupProvider googleSecurityGroupProvider
 
   CopyLastGoogleServerGroupAtomicOperation(BasicGoogleDeployDescription description) {
```

Comment:
I can't write comments on the code that wasn't changed ,but some of your messages omit `newDescription.details`